pkg/controllers/localbuild/controller.go

```diff
@@ -278,9 +279,123 @@ func (r *LocalbuildReconciler) SetupWithManager(mgr ctrl.Manager) error {
 }
 
 func (r *LocalbuildReconciler) ReconcileArgoAppsWithGitea(ctx context.Context, req ctrl.Request, resource *v1alpha1.Localbuild) (ctrl.Result, error) {
-	log := log.FromContext(ctx)
+	logger := log.FromContext(ctx)
```

Comment:
Problem with using `log` is that it collides with package import like this https://github.com/cnoe-io/idpbuilder/blob/09608f07787012836c6d57eda4b691eefb8ed283/pkg/controllers/localbuild/controller.go#L21 

I agree that we should use logger instead of log across the project. I'll file an issue.